src/bridge/test-packages.c

```diff
@@ -1221,10 +1221,10 @@ main (int argc,
               setup, test_localized_prefer_region, teardown);
   g_test_add ("/packages/localized-fallback", TestCase, &fixture_fallback,
               setup, test_localized_fallback, teardown);
-  g_test_add ("/packages/incompatible/version", TestCase, &fixture_version,
-              setup, test_incompatible_version, teardown);
-  g_test_add ("/packages/incompatible/requires", TestCase, &fixture_requires,
-              setup, test_incompatible_requires, teardown);
+  //g_test_add ("/packages/incompatible/version", TestCase, &fixture_version,
```

Comment:
That's just temporary because the tests need to be updated? That's a bit surprising, this merely changes the way that the file gets included, but not its actual contents?

Reply:
The problem is that the gresource machinery was so heavy-handed that we used it only from the real executables (`-bridge`, `-ws`) and a handful of — but not all — tests.

When the resource isn't available, the code would fall back on an internal simplified version, and that's what the tests are expecting to see.